ui/src/components/WorkerManagerWorkerTypesTable/index.jsx

```diff
@@ -0,0 +1,201 @@
+    const sortDirection = this.state.sortBy === sortBy ? toggled : 'desc';
+
+    this.setState({ sortBy, sortDirection });
+  };
+
+  renderRow = workerType => {
+    const { path } = this.props;
+    const iconSize = 16;
+    const { failedNumber, exceptionNumber, unscheduledNumber } = workerType;
+
+    return (
+      <TableRow key={workerType.name}>
+        <TableCell>
+          <TableCellListItem
+            button
+            component={Link}
+            to={`${path}/worker-types/${workerType.name}`}>
+            <ListItemText
+              disableTypography
+              primary={<Typography>{workerType.name}</Typography>}
+            />
+            <LinkIcon size={iconSize} />
+          </TableCellListItem>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.pendingCapacity}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.runningCapacity}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.pendingTasks}</Typography>
+        </TableCell>
+
+        <CopyToClipboard
+          title={`${workerType.lastActive} (Copy)`}
+          text={workerType.lastActive}>
+          <TableCell>
+            <TableCellListItem button>
+              <ListItemText
+                disableTypography
+                primary={
+                  <Typography>
+                    <DateDistance from={workerType.lastActive} />
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellListItem>
+          </TableCell>
+        </CopyToClipboard>
+
+        <CopyToClipboard
+          title={`${workerType.lastResolved} (Copy)`}
+          text={workerType.lastResolved}>
+          <TableCell>
+            <TableCellListItem button>
+              <ListItemText
+                disableTypography
+                primary={
+                  <Typography>
+                    <DateDistance from={workerType.lastResolved} />
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellListItem>
+          </TableCell>
+        </CopyToClipboard>
+
+        <TableCell>
+          <Typography>{failedNumber}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{exceptionNumber}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{unscheduledNumber}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <TableCellListItem
+            button
+            component={Link}
+            to={`${path}/providers/${workerType.provider}`}>
+            <ListItemText
+              disableTypography
+              primary={<Typography>{workerType.provider}</Typography>}
+            />
+            <LinkIcon size={iconSize} />
+          </TableCellListItem>
+        </TableCell>
```

Comment:
Hovering over an entry of the provider shows:

![Screen Shot 2019-04-23 at 3 53 24 PM](https://user-images.githubusercontent.com/3766511/56611545-06a3f980-65e0-11e9-8f6d-55ccd4598202.png)

![Screen Shot 2019-04-23 at 3 53 16 PM](https://user-images.githubusercontent.com/3766511/56611544-06a3f980-65e0-11e9-9dc1-21049396ebe4.png)

Reply:
done